packages/viz/src/lib/basemap/create-google-map.ts

```diff
@@ -0,0 +1,51 @@
+import { CartoBaseMapError } from '../errors/basemap-error';
+
+const DEFAULT_OPTIONS: google.maps.MapOptions = {
+  mapTypeId: 'roadmap',
+  center: { lat: 0, lng: 0 },
+  zoom: 1
+};
+
+/**
+ * A helper function to create a GoogleMaps basemap on a 'map' DOM element, rendered using *Google Maps JS API*
+ *
+ * Examples:
+ * ```javascript
+ *    // Several options to create the map are allowed
+ *    const deckMap = carto.viz.createGoogleMap();
+ *    const deckMap = carto.viz.createGoogleMap('map', { mapTypeId: 'satellite'});
+ *    const deckMap = carto.viz.createGoogleMap('map', { mapTypeId: 'hybrid', zoom: 4 });
+ *    const deckMap = carto.viz.createGoogleMap('map', { mapTypeId: 'terrain', zoom: 4, center: { lng: 3, lat: 40 } });
+ * ```
+ * @export
+ * @param {Element | string} container
+ * @param {google.maps.MapOptions} googleMapsOptions
+ * @returns
+ */
+export function createGoogleMap(
```

Comment:
I find the new API definitions much better than the older ones!

One thing that might be nit-picking but would be great is to define a common API parameter definition across both type of maps.

Let me explain, both types of maps have some common attributes like the center, zoom, and so on. And then some specific parameters like `mapTypeId` in this case.

So, why don't we change this API specification to have a common specification for common parameters so that developers just need to change the method if they want to convert their CARTO map to a Google Maps map?

Reply:
Yeah, I thought about that, but I finally decided to keep each other's interface for simplicity, without any abstraction. I think that conversion is out of scope right now.